datetime_test.go

```diff
@@ -82,6 +84,17 @@ func TestDayOfWeek(t *testing.T) {
 	}
 }
 
+func TestDayOfWeekReturnsDifferentValues(t *testing.T) {
+	dayMap := make(map[string]struct{})
+	iterations := 5 // sufficiently large to assure we don't randomly get the same value again
+	for i := 0; i < iterations; i++ {
+		day := GetDateTimer().DayOfWeek()
+		//t.Log(day)
+		dayMap[day] = struct{}{}
+	}
+	assert.True(t, len(dayMap) > 1)
+}
+
```

Comment:
Because you already using map, I think you can also utilize the map to avoid the same value repeated. 
```
day := GetDateTimer().DayOfWeek()
if _, ok := dayMap[day]; ok {
	i--
	continue
}
dayMap[day] = struct{}{}
t.Log(day) // Will print random day 5 times. 
```

Reply:
Because you already using map, I think you can also utilize the map to avoid the same value repeated.
```go
day := GetDateTimer().DayOfWeek()
if _, ok := dayMap[day]; ok {
	i--
	continue
}
dayMap[day] = struct{}{}
t.Log(day) // Will print random day 5 times. 
```